### ISSUE TITLE
FUNCTION_SOURCE must be absolute

### ISSUE THREAD

**Issue by grant** [GoogleCloudPlatform/functions-framework-php#15]
FUNCTION_SOURCE must be absolute.
This is different than the other FFs and should be documented. Or there should be a better warning for when the file can't be found.

```
[error] 30#30: *2 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught RuntimeException: Unable to load function from "index.php" in /srv/vendor/google/cloud-functions-framework/router.php:35
```

It has to be relative from the `router.php` file.

**Comment by bshaffer** [comment]
What is the behavior of the other function frameworks?

**Comment by grant** [comment]
Node defualts path to cwd and accepts abs or rel path.
Go doesn't support this feature.

I suggest we default to a path that allows for a root index.php to just work.

**Comment by bshaffer** [comment]
The default IS to use a root `index.php`. The error only occurs if you supply a manual relative path.

**Comment by bshaffer** [comment]
For the Node paths, are they relative to the project root?

**Comment by grant** [comment]
> For the Node paths, are they relative to the project root (e.g. `/srv`)?

Node uses a path resolver such that paths such as `/root/test/`,  `.`, `./foo/bar` are resolved to an absolute path internally. This allows you to use either relative or absolute paths for the `FUNCTION_SOURCE` configuration.

Here are more details:

- https://nodejs.org/api/path.html#path_path_resolve_paths
- https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/0b348e8e643c5d7212e6a4812f203ba3812ff159/src/index.ts#L64

**Comment by bshaffer** [comment]
Fixed in #20
